other-builtin-functions/03_current_watermark/03_current_watermark.md

```diff
@@ -0,0 +1,43 @@
+# 03 Filter out late data
+
+![Twitter Badge](https://img.shields.io/badge/Flink%20Version-1.14%2B-lightgrey)
+
+> :bulb: This example will show how to filter out late data using the `CURRENT_WATERMARK` function.
+
+The source table (`mobile_usage`) is backed by the [`faker` connector](https://flink-packages.org/packages/flink-faker), which continuously generates rows in memory based on Java Faker expressions.
+
+As explained before in the [watermarks recipe](../../aggregations-and-analytics/02_watermarks/02_watermarks.md), Flink uses watermarks to measure progress in event time. By using a `WATERMARK` attribute in a table's DDL, we signify a column as the table's event time attribute and tell Flink how out of order we expect our data.  
+
+There are many cases when data is arriving out of order and late. An example could be when someone is using a mobile app while being offline because of lack of mobile coverage or flight mode being enabled. When Internet access is restored, previously tracked activities would then be sent.
+
+In this recipe, we'll filter out this late data using the [`CURRENT_WATERMARK`](https://ci.apache.org/projects/flink/flink-docs-release-1.14/docs/dev/table/functions/systemfunctions/) function. You could then send this data to a different sink, because you might want to treat this data differently in your downstream consumer, like a product recommender. 
+
+This table DDL contains both an event time and a processing time definition. `ingest_time` is defined as processing time, while `log_time` is defined as event time and will contain timestamps between 45 and 10 seconds ago.   
```

Comment:
Is the ingestion_time important here or just "noise"?

Reply:
I deliberately added ingestion_time to have a visual representation of processing time, event time and watermarks